model/src/main/java/uk/gov/service/payments/commons/model/CardExpiryDate.java

```diff
@@ -25,7 +25,13 @@ private CardExpiryDate(String expiryDate) {
         this.month2Digits = matcher.group(1);
         this.year2Digits = matcher.group(2);
     }
-
+    
+    private CardExpiryDate(YearMonth expiryDate) {
+        int month = expiryDate.getMonthValue();
+        this.month2Digits = (month < 10) ? ("0" + month) : String.valueOf(month);
+        this.year2Digits = String.valueOf(expiryDate.getYear()).substring(2,4);
```

Comment:
This only works if it’s a four-digit year (`YearMonth` supports up to nine-digit years for when you want to make plans for the far future!) but the validation will ensure this is the case.